README.md

```diff
@@ -2,22 +2,135 @@
+This repo contains packages that are in maintenance and primarily updated for security servicing, as opposed to new features. They were previously hosted in repos or branches that are no longer active.
 
-## Project FAQ
+# How do I know if a particular package belongs on this repo?
 
-- How do I know if a particular package belongs on this repo?
-  - As explained above, this repo is meant to be the home for packages that are no longer building in any supported repository/branch. Therefore, if a package/library is still building in one supported branch, then this repository is not the right place for it. The idea of this repo is to have a servicing vehicle for those packages, and if one package is still building in a supported branch, then by definition that is the servicing vehicle that should be used for servicing fixes to it.
-- Ok, I've determined that this package does belong on the repo, what is the process to add it?
-  - *ToDo*
+If a package is still building in a supported branch, then this repository is not the right place for it, and the supported branch where they're still building is their preferred servicing vehicle.
 
-## How to produce stable package versions
+On the other hand, if the package is still targeting frameworks that are still in support, and contains an actual source code implementation (as opposed to being a reference assembly or containing only type-forwards), but its source code currently lives in a branch from a .NET version that is already out of support, then the package is a candidate for getting migrated to this repo.
 
-Package versions produced by official builds for this repo will automatically contain a prerelease suffix. This is in order to be able to iterate through changes until we are ready to have a final build when we are ready for a new servicing release. When that time comes, all that is needed in order to produce packages that don't have the prerelease suffix is to manually queue a build in the official pipeline, and set the variable `DotNetFinalVersionKind` to `release`. This will automatically cause Arcade to set the right version for the package, as well as use a dedicated NuGet feed to push the final build assets (in order to avoid potential version clashes).
+## How to migrate a library
+
+1. You can use the `eng/Migrate-Package.ps1` script to automate most of the migration work, which includes most commit history relevant to the chosen assembly. Depending on the repo and branches of origin, as well as the root for all your cloned repos, you'll have to adjust the arguments. For example:
+
+    a. Migrating a package from .NET Core 1.1:
+   ```powershell
+   .\eng\Migrate-Package.ps1 \
+      -OriginRepoPath D:/corefx \
+      -OriginRemote upstream \
+      -OriginBranch v1.1.13 \
+      -AssemblyFileOrDirectoryToMigrate System.Migratable.Assembly \
+      -AssemblyFileOrDirectoryRelativeDirectoryPath src \
+      -DestinationRepoPath D:/maintenance-packages
+    ```
```

Comment:
I'm relatively sure that we will never bring source code from corefx/1.1.x over. I would remove that entire block:

```suggestion
```

Reply:
Do you mean besides [System.Xml.XPath.XmlDocument](https://github.com/dotnet/maintenance-packages/releases/tag/2024.11.12), we won't migrate anything else from that release?

Reply:
Oh I didn't know that we brought a package from 1.1 over. Interesting. Disregard my comment then. I wasn't expecting that to just work.